students/eowyn/session02/print_grid.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+''' Construct grid that is composed of ndim boxes in 
+    either direction, and each box is composed of ndash 
+    lines both horizontally and vertically '''
+
+### First method uses global vars and functions for lines and verts
+### then a caller function to print it all
+
+# define global varaibles
+plus = "+"
+minus = "-"
+space = " "
+dash = '|'
+
+def make_line(ndim,ndash):
```

Comment:
another style point -- better to put a space after a comma: make_line(ndim, ndash).